gp-includes/cli/translation-set.php

```diff
@@ -7,7 +7,7 @@ class GP_CLI_Translation_Set extends WP_CLI_Command {
 	 * @param string $project Project path
 	 * @param string $locale Locale slug
 	 * @param string $set Set slug
-	 * @return GP_Translation_Set|WP_Error Translation set if available, error otherwise.
+	 * @return object Translation set (GP_Translation_Set class) if available, error (WP_Error class) otherwise.  Note, PHPDoc doesn't know about WP_Error so we can explicitly define it in the return types.
```

Comment:
>  and let tools work with the codebase in a consistent manner.

True, but it's a plugin for WordPress. Maybe there is a way to include WordPress as a library? Another example is `WP_Query`.